spanner/api/Spanner/Program.cs

```diff
@@ -1888,6 +1943,267 @@ await cmd.ExecuteReaderAsync().ConfigureAwait(false))
             }
         }
 
+        public static object InsertStructSampleData(string projectId,
```

Comment:
nit: either put all the parameters on the same line (they should fit IMO) or put every parameter on a separate line)